src/slider/slider.scss

```diff
@@ -40,6 +40,7 @@ $tick-height: $iui-baseline; // 11px
   position: relative;
   flex-grow: 1;
   cursor: pointer;
+  touch-action: pan-y;
```

Comment:
Thats interesting.. I thought y is vertical, while the slider is horizontal....

Reply:
pan-y means the browser will preserve the vertical panning behavior (and disable default handling of pan-x, pinch-zoom, and other touch actions). This allows us to use the slider without the browser touch actions getting in the way.

I could have also gone with `touch-action: none` and it would work but that would be more restrictive.

Reply:
Well, maybe I am not that good reader of the docs..
![image](https://user-images.githubusercontent.com/81580355/125803476-1be13000-c9a3-432a-b64e-779f3d63821d.png)

Reply:
Maybe the docs are not clear, but `pan-y` means "browser should handle vertical panning, user should handle rest".